src/compiler/typescript_to_rust/generators/azle_into_js_value_trait.ts

```diff
@@ -167,5 +167,11 @@ export function generateAzleIntoJsValueTrait(): Rust {
                 boa_engine::JsValue::BigInt(boa_engine::bigint::JsBigInt::from_string(&self.0.to_string()).unwrap())
             }
         }
+
+        impl AzleIntoJsValue for ic_cdk::export::candid::Reserved {
+            fn azle_into_js_value(self, _: &mut boa_engine::Context) -> boa_engine::JsValue {
+                boa_engine::JsValue::Undefined
```

Comment:
Should this be `Null` instead of `Undefined`?

Reply:
Let's change it to Null

Reply:
Done